### DISCUSSION TITLE
Ajuda com o Benefício de prova GitHub Certifications Foundation

### DISCUSSION THREAD

**Discussion by Leonardo-Talanski** [community/community#140214] [GitHub Education] [answered]
### Hi everyone,

Olá Comunidade, faço parte Global Campus como estudante e estou tentando conseguir o benefício de fazer a prova do github foundations gratuitamente e quando vou no check-out do site da PSI o valor não é alterado para 0 ou seja gratuito. 
   Mais alguém com esse problema ou que tenha a solução ?

![checkout](https://github.com/user-attachments/assets/860953fc-a0a1-4087-bed8-9626fa7f48c6)

**Comment by davevad93** [comment, marked as answer]
Hi @Leonardo-Talanski ,

There was an error and they have temporarily disabled the certification offer, you can stay tuned by subscribing to this discussion #138834, they will update us when it is fixed.

This is the latest update from a couple of hours ago:

> Hi All - Thanks so much for your patience as we work across multiple teams to get this offer back up and running for students. Estimates are that the offer will resume by the end of October. Keep checking back here for any updates. Thanks!

**Reply by Leonardo-Talanski** [reply]
Thanks Davide.

**Reply by davevad93** [reply]
You're welcome @Leonardo-Talanski 👍
